services/src/main/scala/cromwell/services/metadata/WorkflowQueryKey.scala

```diff
@@ -127,9 +128,14 @@ object WorkflowQueryKey {
       val nels: List[ErrorOr[String]] = values map { v => {
         allowedValues.contains(v).fold(v.validNel[String], v.invalidNel[String])
       }}
-      sequenceListOfValidatedNels("Unrecognized values", nels)
+      sequenceListOfValidatedNels(s"Keys should be from ${allowedValues.toString}. Unrecognized values", nels)
     }
   }
+
+  case object IncludeSubworkflows extends BooleanWorkflowQueryKey[Boolean] {
+    override val name = "Includesubworkflows"
```

Comment:
`Includesubworkflows` or `includeSubworkflows`? Or does it not matter?

Reply:
It needs to be `Includesubworkflows` as there is validation logic for recognizing the valid keys (inside WorkflowQueryParameters class) where it converts them to lowercase and capitalizes the first letter